models/patient.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
+"""Module for Patient class."""
+
+from models.base_user import BaseUser
+from datetime import date
+
+
+class Patient(BaseUser):
+    """Class representing a Patient."""
+    hospital_number = ""
+    consult_id = []
```

Comment:
pluralize all the ids and change hospital_number to **pid**(short for patient id)